bcp47/library/Data/BCP47/Trie/Internal.hs

```diff
@@ -9,6 +9,7 @@ module Data.BCP47.Trie.Internal
   , union
   , unionWith
   , unionUsing
+  , mapMaybe
```

Comment:
You'll need to export this from the `Trie.hs` module. This is an internal module.